examples/tensorflow/segmentation/evaluation.py

```diff
@@ -35,22 +35,14 @@
 
 
 def get_argument_parser():
-    parser = get_common_argument_parser(weights=False,
+    parser = get_common_argument_parser(mode=True,
+                                        weights=False,
                                         epochs=False,
                                         precision=False,
                                         save_checkpoint_freq=False,
                                         to_h5=False,
                                         dataset_type=False)
 
-    parser.add_argument(
-        '--mode',
-        '-m',
-        nargs='+',
-        choices=['train', 'test', 'export'],
-        default='train',
-        help='train: performs validation during training; test: tests the model; export: exports the model.'
```

Comment:
You changed the description string from 'train: performs validation during training; test: tests the model; export: exports the model.' to `train: performs training and validation; test: tests the model; export: exports the model.` which is less informative.

Reply:
Reworded to be more explicit.